packages/web-app/src/views/questionnaire-check-and-confirm.njk

```diff
@@ -41,7 +60,7 @@
     value: { text: questionnaireData.localPlanningAuthorityName }
 } %}
 
-{% if reviewOutcome === '2' %}
+{% if reviewOutcome === 'Incomplete' %}
```

Comment:
Why has this been changed? Or we mapping this somewhere?

Reply:
This is the value that is being passed from review-questionnaire-submission page. This is how- Incomplete / Complete - it was developed originally. But recent change - not by me - changed this to 1, 2 similar to they do in Team 1.

Reply:
I imagine the reason why it was changed to a numeric value is too map to the PK of the LookUpQuestionnaireOutcome table. Might be worth reaching out to Team 1 on confirming why do they do it in the first place so we can be consistent across the board otherwise we risk breaking things unnecessarily.

Reply:
I agree with @demzzydev, this value shouldn't be changed.

Reply:
If the form is passing `Incomplete` as a value then that's the value that needs to be changed

Reply:
Reverted the changes as per PR comments